cupyx/scipy/sparse/linalg/_eigen.py

```diff
@@ -344,7 +344,7 @@ def svds(a, k=6, *, ncv=None, tol=0, which='LM', maxiter=None,
     cond = factor[t] * numpy.finfo(t).eps
     cutoff = cond * cupy.max(w)
     above_cutoff = (w > cutoff)
-    n_large = above_cutoff.sum()
+    n_large = above_cutoff.sum().item()
```

Comment:
This seems to be introducing device synchronization. But I guess there is no easy workaround sot this is acceptable.